src/eradiate/experiments/_core.py

```diff
@@ -477,27 +477,45 @@ def process(
             measure.sensor_id: measure for measure in self.measures
         }
 
+        def convert_to_Y_format(img):
```

Comment:
Naming convention: let's stick to `snake_case`.
```suggestion
        def convert_to_y_format(img):
```